newa/__init__.py

```diff
@@ -106,6 +108,52 @@ def render_template(
         raise Exception("Could not render template.") from exc
 
 
+@define
+class Settings:
+    """ Class storing newa settings """
+
+    et_url: Optional[str] = None
+    rp_url: Optional[str] = None
+    rp_token: Optional[str] = None
+    rp_project: Optional[str] = None
+    jira_url: Optional[str] = None
+    jira_token: Optional[str] = None
+    jira_project: Optional[str] = None
+    tf_token: Optional[str] = None
+    tf_recheck_delay: Optional[str] = None
+
+    def get(self, key: str, default: Optional[str] = '') -> str:
+        return str(getattr(self, key, default))
+
+    @classmethod
+    def load(cls: type[SettingsT], config_file: Path) -> Settings:
+        cp = ConfigParser()
+        cp.read(config_file)
+
+        def __get(
+                cp: ConfigParser,
+                path: str,
+                envvar: Optional[str] = None,
```

Comment:
It seems that all `__get()` calls specify `envvar`, `str` and no default value would reflect the situation better.

Reply:
I thought there might be options where envvar override won't be available but currently there aren't any and maybe there won't be ever. I have changed it.